ksrates/setup_correction.py

```diff
@@ -19,13 +19,18 @@ def setup_correction(config_file, nextflow_flag):
     # Check configfile
     species_of_interest = config.get_species()
     original_tree = config.get_newick_tree()
-    tree = fcTree.reorder_tree_leaves(original_tree, species_of_interest)  # species of interest is the top leaf
+    tree = fcTree.reorder_tree_leaves(original_tree, species_of_interest)  # focal species is the top leaf
+    latin_names = config.get_latin_names()
+
+    trigger_exit = False # will trigger exit if FASTA or GFF files are missing or empty
     fasta_dict = config.get_fasta_dict()
     gff_dict = config.get_gff_dict(warn_empty_dict=False)
-    latin_names = config.get_latin_names()
-    if latin_names == {}:
-        logging.error("Exiting")
-        sys.exit(1)
+    # If a GFF is provided, check existence and content
+    if species_of_interest in gff_dict:
+        gff = config.get_gff_name(gff_dict, species_of_interest)
```

Comment:
Why a dict? It can be only one file.

Reply:
See an earlier comment. I don't think the analogy here warrants the usage of a dict, seems unnecessarily convoluted.